modules/boulder/src/main/ts/ephox/boulder/api/FieldPresence.ts

```diff
@@ -1,65 +1,38 @@
-import { Adt, Fun } from '@ephox/katamari';
+import { Fun } from '@ephox/katamari';
```

Comment:
Actual ADT conversion here (everything above this should just be renaming things)